### ISSUE TITLE
Results for WebFace42M

### ISSUE THREAD

**Issue by mnikitin** [DanJun6737/TransFace#1]
Hello,

Really nice work!

But why do you not provide any experimental results using WebFace42M as the training dataset?
It is more "large-scale" dataset compared to Glint360K and that's why it is more natural to train ViT-based architectures using it.

Thank you!

**Comment by DanJun6737** [comment]
Hi mnikitin~

Thanks for your recognition of our work!

Due to our current limited resources, we have temporarily not conducted experiments on the WF42M dataset. We will add this experiment in the future. 

Best wishes!

**Comment by TranThanh96** [comment]
any update on this? I tried with wf42m but not so good

**Comment by DanJun6737** [comment]
Hi @TranThanh96 ~
Recently, we are conducting experiments on the wf42m dataset. It is worth mentioning that ViT is highly sensitive to some hyperparameters such as learning rate. We will upload the pre-trained models and configuration files in the future. Please be patient and thank you for your interest in our work.
